### ISSUE TITLE
Korean locale

### ISSUE THREAD

**Issue by infinite0x20** [wordplaydev/wordplay#475]
## What's the problem?

-  Adding language support for Korean
-  Some work has already been done on the [korean-locale](https://github.com/wordplaydev/wordplay/tree/korean-locale) branch (I'm currently still investigating the branch to figure out the scope of the prior work, along with what still needs to be done for deployment)

## Who benefits?
Korean speakers, and anyone interested in learning/practicing the Korean language!

## Where can we work on this issue?
On the existing [korean-locale](https://github.com/wordplaydev/wordplay/tree/korean-locale) branch, and through this thread. : )

**Comment by anhourandaquarter** [comment]
Pull request #453 updated the branch so that we can begin work on it. I am still going through the updated work to figure out next steps. Will update here when I've got a summary of tasks!

**Comment by anhourandaquarter** [comment]
Looks like we can pick up the translation task from line 1451 in the [korean-ko.json](static/locales/ko-KR/ko-KR.json). (the entry for `Text`). Will be working from here.

**Comment by amyjko** [comment]
I implemented a feature to use Google Translate to generate draft translations. I recently merged Korean drafts into. They should be a good starting point for editing!

**Comment by anhourandaquarter** [comment]
Oh, nice! I'll look into those, thanks!

**Comment by joonpiter** [comment]
@anhourandaquarter will you be going forward with this issues?

**Comment by anhourandaquarter** [comment]
@joonpiter Yes, I'll be working on it during the school year, if that's all right!

**Comment by anhourandaquarter** [comment]
I've created a fork here: https://github.com/anhourandaquarter/wordplay/tree/korean-locale

I'll be working on the `korean-locale` branch inside that fork and submitting pull requests there.

**Comment by anhourandaquarter** [comment]
End-of-quarter check-in-- I'm still working on this issue but will be taking a short hiatus during winter break. I'll submit a pull request with the most up-to-date changes, and resume working when Winter Quarter starts

**Comment by anhourandaquarter** [comment]
Hi @noahp44 sure thing! I've just added you to the current fork. @EugeneLim0805 and I have been working on verifying and editing the machine translation of `ko-KR.json`. Let me know if you have any questions on what to do / how to get started. : )

**Comment by amyjko** [comment]
Hi! I'm Amy's GitHub bot. @infinite0x20 — Are you still working on this issue? Can you post a status update, and if you won't finish it, can you unassign yourself? We'll unassign you in a week if we don't get a reply.

**Comment by infinite0x20** [comment]
Hello! Yes, I'm still working on this issue. There have been a few delays in getting a pushable update because I want to make sure that the language is appropriate for middle school, but I don't have a great frame of reference. Would love to find others who are interested in working on this localization so that the translation is accurate and audience-appropriate!

**Comment by amyjko** [comment]
Hi! I'm Amy's GitHub bot. @infinite0x20 — Are you still working on this issue? Can you post a status update, and if you won't finish it, can you unassign yourself? We'll unassign you in a week if we don't get a reply.

**Comment by infinite0x20** [comment]
Yes, still working on this!

**Comment by amyjko** [comment]
Hi! I'm Amy's GitHub bot. @infinite0x20 — Are you still working on this issue? Can you post a status update, and if you won't finish it, can you unassign yourself? We'll unassign you in a week if we don't get a reply.
